### ISSUE TITLE
Input for background shouldn't be case-sensitive

### ISSUE THREAD

**Issue by QuincyLarson** [FreeCodeCamp/hour-of-code#31]
![](https://www.evernote.com/l/Ar--D75IAbdGxZ2nvyhJZMgnpRt004xNP68B/image.png)

**Comment by Daynil** [comment]
This appears to already be case insensitive.

**Comment by QuincyLarson** [comment]
@Daynil this isn't. I did a hard refresh and attempted it again, and "Stone" isn't working, but "stone" is.

![hour_of_code_by_free_code_camp_](https://cloud.githubusercontent.com/assets/985197/20873356/3a262836-ba6c-11e6-9fec-b05f011c40e4.png)

**Comment by Daynil** [comment]
I did some research and I think this is an OS-level difference. I think Cindy and I are windows so we couldn't replicate the issue. If I recall, you're on mac/linux? 

The only way to make this challenge 100% insensitive to case is to change it from a direct style modification to a function call. I changed the wording and the seed code to reflect the change in #32 , let me know if this  looks good on your side @QuincyLarson.
